src/call.cpp

```diff
@@ -2201,7 +2234,7 @@ char* call::createSendingMessage(SendingMessage *src, int P_index, char *msg_buf
             }
             break;
         case E_Message_Last_Request_URI: {
-            char * last_request_uri = get_last_request_uri();
+            char * last_request_uri = get_last_header_uri("To:");
             dest += sprintf(dest, "%s", last_request_uri);
             free(last_request_uri);
```

Comment:
Why not this?
```
dest += sprintf(dest, "%s", get_last_header("To:uri"));
```
And drop the entire `get_last_header_uri()` method.

The get_uri-code could be implemented in `sip_parser.cpp` instead as `get_first_uri(const char *msg, const char *name, const char *shortname)` or something, with the same semantics as the other get_* functions.